python/ray/tests/test_failure.py

```diff
@@ -82,6 +82,38 @@ def f():
         assert False
 
 
+# Note that `ray_start_2_cpus` is required here. If we use
```

Comment:
This sounds like it's bound to be flaky. Could you come up with a deterministic way to force this to happen? e.g., by having the two tasks take in an actor and signal that actor that they're running before proceeding with the test?

Reply:
Moved these functions into an actor. And we don't need to check if the actor is alive before submitting tasks because an actor only needs one worker.